core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceUnitTests.scala

```diff
@@ -391,7 +391,13 @@ class WorkspaceServiceUnitTests extends AnyFlatSpec with OptionValues with Mocki
     val applicationBinding = new RoleBinding().role(IamRole.APPLICATION).members(List("application@example.com").asJava)
     val wsmRoleBindings = new RoleBindingList()
     wsmRoleBindings.addAll(
-      List(ownerBinding, writerBinding, readerBinding, discovererBinding, applicationBinding).asJava
+      List(projectOwnerBinding,
```

Comment:
Adding the project owner here caused the existing `fetch policies from WSM for McWorkspaces` test to fail without the fix.